.travis.yml

```diff
@@ -32,17 +32,20 @@ jobs:
       before_script: echo "$DOCKER_PASSWORD" | docker login --username "$DOCKER_USERNAME" --password-stdin ${docker_url}
       script:
         - docker build -f compose/master/Dockerfile -t ${docker_url}/${docker_repository}/mlbench_master:travis-ci-test .
-        - docker build -f compose/worker/Dockerfile -t ${docker_url}/${docker_repository}/mlbench_worker:travis-ci-test .
+        - docker pull liehe/mlbench:mlbench-worker-dev && docker build -f compose/worker/Dockerfile -t ${docker_url}/${docker_repository}/mlbench_worker:travis-ci-test .
```

Comment:
I added you to the mlbench repo, then we can add the image to mlbench instead of liehe.

And I named the images wrong, it seems kebab case (mlbench-worker-dev) is the default for docker images over Snake case (mlbench_worker_dev). so we can rename the other images as well. 

I think for now it's ok, though, and that's something we can do at a later date. See #30